tracardi/service/storage/drivers/elastic/event.py

```diff
@@ -220,3 +220,19 @@ def convert_data(raw_result):
                         yield record
 
     return list(convert_data(raw_result))
+
+
+async def update_fields(event_type: str, field: str, value: Any):
```

Comment:
This update field not fields. Change name of method to update_field.

Reply:
done